### PR TITLE
add childrenSortBy and index to data-object and data-object-folder

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

File: src/GraphQL/Resolver/DataObject.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\GraphQL\Resolver;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Service;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Traits\ServiceTrait;
+
+class DataObject
```
Comment (weisswurstkanone) [inline src/GraphQL/Resolver/DataObject.php:22]: does it make sense to extend from Element resolver ?

File: src/GraphQL/Resolver/DataObject.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+{
+    use ServiceTrait;
+
+    public function __construct(Service $graphQlService)
+    {
+        $this->setGraphQLService($graphQlService);
+    }
+
+    /**
+     * @param array            $value
```
Comment (weisswurstkanone) [inline src/GraphQL/Resolver/DataObject.php:32]: this is probably not an array (see how it is used in getById)

File: src/GraphQL/Resolver/DataObject.php (hunk 1 above)
Comment (dpfaffenbauer) [reply src/GraphQL/Resolver/DataObject.php:22]: not sure honestly, if you want me to extend from it, I can change it

Comment (weisswurstkanone) [reply src/GraphQL/Resolver/DataObject.php:22]: I would vote for it, but this is probably a matter of taste

Comment (dpfaffenbauer) [reply src/GraphQL/Resolver/DataObject.php]: done

File: src/GraphQL/Resolver/DataObject.php (hunk 2 above)
Comment (dpfaffenbauer) [reply src/GraphQL/Resolver/DataObject.php:32]: done

**Comment by weisswurstkanone** [comment]
thanks a lot!

**Pull request merged**